go.mod

```diff
@@ -2,34 +2,36 @@ module github.com/minio/mc
 
 go 1.13
 
+replace github.com/minio/minio => ../minio
```

Comment:
Could you remove replace tag in this file ?

Reply:
this is needed until minio PR is merged